consensus/src/main/java/org/tron/consensus/dpos/DposService.java

```diff
@@ -27,6 +24,8 @@
 import org.tron.core.capsule.BlockCapsule;
 import org.tron.core.capsule.WitnessCapsule;
 
+import static org.tron.core.config.Parameter.ChainConstant.*;
```

Comment:
check style do not allow *

Reply:
OK, but the local check style is passed.